static/app/components/slideOverPanel.tsx

```diff
@@ -45,25 +45,24 @@ function SlideOverPanel(
     ? COLLAPSED_STYLES[slidePosition]
     : COLLAPSED_STYLES.right;
 
-  return (
-    <AnimatePresence>
-      {!collapsed && (
-        <_SlideOverPanel
-          ref={ref}
-          initial={collapsedStyle}
-          animate={openStyle}
-          exit={collapsedStyle}
-          slidePosition={slidePosition}
-          transition={{
-            type: 'spring',
-            stiffness: 500,
-            damping: 50,
-          }}
-        >
-          {children}
-        </_SlideOverPanel>
-      )}
-    </AnimatePresence>
+  return collapsed ? null : (
+    <_SlideOverPanel
+      ref={ref}
+      initial={collapsedStyle}
+      animate={openStyle}
+      exit={collapsedStyle}
+      slidePosition={slidePosition}
+      transition={{
+        type: 'spring',
+        stiffness: 500,
+        damping: 50,
+      }}
+      role="complementary"
+      aria-hidden={collapsed}
+      aria-label="slide out drawer"
```

Comment:
With a role of `"complementary"` this label is redundant. Would be nicer for screen readers to allow `ariaLabel` as a prop, so we can pass descriptions like "Query Samples" and the reader would read out something like "page landmark, complementary, Query Samples" rather than something like "page landmark, complementary, slide out drawer"

Reply:
👍 to a label defined by the options